challenges/2021_cloud_base_height/01_send_data_to_zarr.ipynb

```diff
@@ -0,0 +1,4235 @@
+{
```

Comment:
<p>It might be nice to include a concluding cell to the notebook to sum up what has been done, and maybe how things could be improved further given tme, and maybe a summary of links to relevant reading material?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MetOffice/data_science_cop/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='83'/>